app/fissile.go

```diff
@@ -1224,18 +1215,10 @@ func (f *Fissile) GraphEdge(fromNode, toNode string, attrs map[string]string) er
 }
 
 // Validate runs all checks against all inputs
-func (f *Fissile) Validate(lightManifestPath, darkManifestPath string, defaultFiles []string) error {
+func (f *Fissile) Validate(lightManifestPath, darkManifestPath string) error {
 	var defaultsFromEnvFiles map[string]string
```

Comment:
I think we could also get rid of this variable and change `f.validateManifestAndOpinions` to not take this argument anymore, no? And remove `f.checkEnvFileVariables` because it's then no longer used anywhere.